sources/academy/platform/getting_started/creating_actors.md

```diff
@@ -1,131 +1,180 @@
+title: Creating Actors
+description: Build and run your very first Actor directly in Apify Console from a template. This lesson provides hands-on experience with building and running Actors.
 sidebar_position: 2
 slug: /getting-started/creating-actors
 ---
 
 # Creating actors {#creating-actors}
 
-**This lesson offers a hands-on experience in building and running an Actor on the Apify platform using a template. By the end of it, you will build and run your first Actor using an Actor template.**
+**This lesson offers hands-on experience in building and running Actors in Apify Console using a template. By the end of it, you will be able to build and run your first Actor using an Actor template.**
 
 ---
 
-There are several ways to create an Actor, but the easiest method is by selecting an Actor development template. We do not need to install any special software, and everything can be done directly on the Apify platform using an Apify account.
+There are several ways to create an Actor. You can create one from your own source code hosted in a Git repository or in your local machine, for example. But in this tutorial, we'll focus on the easiest method: selecting an Actor code template. We don't need to install any special software, and everything can be done directly in Apify Console using an Apify account.
 
-## Choose your template {#choose-your-template}
+## Choose the source {#choose-the-source}
 
-On the Apify Console, in the [Actors tab](https://console.apify.com?asrc=developers_portal), click on the **Create new** button in the top right-hand corner.
+Once you're in Apify Console, go to [Actors](https://console.apify.com/actors?asrc=developers_portal), and click on the **Create an Actor** button in the top right-hand corner.
 
-![Create new button](./images/create-new-actor.png)
+![Create an Actor button](./images/create-new-actor.png)
 
-You will be presented with a page featuring several template options in Python, JavaScript, and TypeScript. You can select the one that best fits your specific use case. For example, let's choose the **Start with JavaScript** template and click on **Let's go**.
+You'll be presented with a page featuring two ways to get started with a new Actor.
 
-> Also feel free to give your Actor a custom name instead of the default one. For this lesson though, we'll keep it as **my-actor**.
+1. Creating an Actor from existing source code (using Git providers or pushing the code from your local machine using Apify CLI)
+2. Creating an Actor from a code template
 
-![Choosing a template](./images/choose-template.png)
+| Existing source code             |  Code templates |
+:-------------------------:|:-------------------------:
+![Create and Actor from source code](./images/create-actor-from-source-code.png)  |  ![Create an Actor from code templates](./images/create-actor-from-templates.png)
 
-## Hello World! {#hello-world}
+## Creating Actor from existing source code {#existing-source-code}
```

Comment:
I am not sure if this section really makes sense here in Academy at this point. I think that this belongs to docs or to some later section in Academy, for this reason -

![CleanShot 2023-08-08 at 11 00 59@2x](https://github.com/apify/apify-docs/assets/594801/cf9d6cc2-b116-4bf3-802e-cb636489744f)

Reply:
I'd just move this redundant info to normal documentation and link it here as a hint "Btw you can also import existing Actor, learn about this here [LINK]" but I would not distract the user from the main point of this article which is a creation of an Actor from the template.

Reply:
I wasn't actually sure about this. The article is called **Creating Actors**, not **Creating Actors from the template** which is only the subset of how to create Actors (on Apify Platform).

What about if you read about Apify Platform and you want to deploy your scraper there and you don't know how... you Google "Apify Create Actor" and you can find this article. I think users would be quite lost without having it there.

I don't have a strong opinion on this but it still makes sense to me.

What do you think @dodaniel ?

Reply:
I am not very opinionated about this so feel free to do it the way you want. Just try to think about Academy this way:

> Do not think about Academy as about the docs or a reference. It's none of these. It does not need to describe every possibility or feature as documentation. This mainly needs to be a super smooth tutorial where you get to the point without any extra overhead you don't need.

Reply:
I'm keeping it there for now and I agree with moving it to some later section in the academy in the future so there would be a special tutorial for creating Actors from existing code.